src/applications/representative/components/search/RepTypeSelector.jsx

```diff
@@ -12,26 +9,27 @@ const RepTypeSelector = ({ onChange }) => {
   return (
     <>
       <div className="rep-type-selector">
-        <VaRadio
+        <va-radio
```

Comment:
What's the difference between these two? I mostly see `VaRadio` used and a mix of `VaRadioOption` vs `va-radio-option`.

Reply:
React bound components are useful when you need to pass event listeners or other  such functions via props.